WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -92,6 +97,7 @@ def sync(self):
     self.pilotVOScriptPath = ops.getValue("Pilot/pilotVOScriptsPath", self.pilotVOScriptPath)
     self.pilotRepoBranch = ops.getValue("Pilot/pilotRepoBranch", self.pilotRepoBranch)
     self.pilotVORepoBranch = ops.getValue("Pilot/pilotVORepoBranch", self.pilotVORepoBranch)
+    self.uploadToWebApp = (ops.getValue("Pilot/uploadToWebApp") != "False")
```

Comment:
We usually allow case insensitive values for boolean flags 1/0, yes/no, true/false. Here you assume only True/False. "no" will be assumed as True.

```suggestion
    self.uploadToWebApp = (ops.getValue("Pilot/uploadToWebApp","").lower() in ("1","yes","true"))
```

Reply:
Do we ? Can't we decide once for all ? :-)

Reply:
It was always like that: https://github.com/DIRACGrid/DIRAC/blob/f33eb1d79e731f721fd2749f65c5c730775a52cb/ConfigurationSystem/private/ConfigurationClient.py#L164 . So the replacement should be simply:

   self.uploadToWebApp = ops.getValue("Pilot/uploadToWebApp",True)

Reply:
ah nice, I did not see this branch in the code ! :)